benchflow-dsl/src/main/scala/cloud/benchflow/dsl/definition/configuration/BenchFlowTestConfiguration.scala

```diff
@@ -12,6 +12,6 @@ import cloud.benchflow.dsl.definition.configuration.workloadexecution.WorkloadEx
 case class BenchFlowTestConfiguration(
   goal: Goal,
   users: Option[Int],
-  workloadExecution: Option[WorkloadExecution],
+  workloadExecution: WorkloadExecution,
   strategy: Option[ExplorationStrategy],
-  terminationCriteria: Option[BenchFlowTestTerminationCriteria])
+  terminationCriteria: BenchFlowTestTerminationCriteria)
```

Comment:
The `YamlProtocol` checks each key-value pair and will throw an exception if something is missing.